test/e2e/run-drizzle.sh

```diff
@@ -4,28 +4,28 @@ set -e
 set -u
 set -o pipefail
 
-npm run ganache:start -- -b 2 >> /dev/null 2>&1 &
-npm_run_ganache_start_pid=$!
+yarn ganache:start -- -b 2 >> /dev/null 2>&1 &
+ganache_start_pid=$!
 sleep 5
 
 pushd "$(mktemp -d)"
-npm install --no-package-lock truffle
-truffle="$(npm bin)/truffle"
+yarn install --no-lockfile truffle
```

Comment:
This seems like something we shouldn't do, but it looks like this test isn't used anymore? Maybe it can be deleted?

Reply:
I think we can delete the Drizzle tests, since we haven't been running them for a while now

Reply:
I've created a separate PR for this: #6855